src/scss/_common.scss

```
@@ -342,6 +342,10 @@
         tab-size: var(--yfm-tab-size-code, inherit);
     }
 
+    a code {
+        text-decoration: underline;
```

I would suggest formatting links according to the transform's general style - without underlines, with their own color and hover effect.

```suggestion
        color: inherit;
```

https://github.com/user-attachments/assets/6cf21d3d-592a-432a-8760-6ec08b82aec9